mnefun/_mnefun.py

```diff
@@ -936,7 +938,7 @@ def fix_eeg_files(p, subjects, structurals=None, dates=None, run_indices=None):
         names = [name for name in raw_names if op.isfile(name)]
         # noinspection PyPep8
         if structurals is not None and structurals[si] is not None and \
-                dates is not None:
+                        dates is not None:
```

Comment:
Did you editor complain about this line? The original should be okay according to PEP8, whereas this should be a violation (not a multiple of 4 indent)